test/Driver/EDMF/edmf_model.jl

```diff
@@ -28,14 +28,14 @@ Base.@kwdef struct SurfaceModel{FT}
     surface_e_int::FT = 300.0*1004.0
     "Surface total specific humidity [kg/kg]"
     surface_q_tot::FT = 0.0016
-    "Surface internal energy flux [m^3/s^3]"
-    e_int_surface_flux::FT = 0.0
-    "Surface total specific humidity flux [m/s*kg/kg]"
-    q_tot_surface_flux::FT = 0.0
-    "Top internal energy flux [m^3/s^3]"
-    e_int_top_flux::FT = 0.0
-    "Top total specific humidity flux [m/s*kg/kg]"
-    q_tot_top_flux::FT = 0.0
+    "surface sensible heat flux [w/m^2]"
```

Comment:
```suggestion
    "surface sensible heat flux `[w/m^2]`"
```
Using brackets are parsed as links by documenter, so this will result in a broken link in the docs if we don't wrap the brackets in back-ticks to enter code mode.